### DISCUSSION TITLE
[Feature Request]: Audio Visualizer

### DISCUSSION THREAD

**Discussion by Croldfish** [ciderapp/Cider#792] [Added Features]
### The feature you would like

Something where you click on the album cover at the top (or at the bottom if you're using another theme) and it pulls up an audio visualizer of the current song. 
The audio visualizer itself would be a graphic of some sort reacting to the sound that is currently playing.
https://youtu.be/WaNQUOqFM44 <--- something like the circle in this collection of songs
https://youtu.be/PzVpbJ5vJ30 <--- or whatever is happening in this

Sorry if it's a bit confusing but I think you get the idea. 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by tva2000hn** [comment]
What version are you using?

**Comment by cryptofyre** [comment]
This feature already exists in the form of a plugin accessible in Cider.
